packages/noya-state/src/reducers/application.ts

```diff
@@ -169,6 +169,17 @@ export type Action =
       swatchId: string | string[],
       name: string | undefined,
     ]
+  | [type: 'setTextColor', value: Sketch.Color]
```

Comment:
Maybe we can move all of these into a `reducers/textStyles.ts`, similar to `reducers/style.ts`? The main difference would be, the `state` we pass in would need to be the `layer` I think.

Reply:
Yes! I'll add them into an unique file !

Reply:
I added it but did it in a way